src/caveexpress/server/entities/Player.cpp

```diff
@@ -590,9 +590,6 @@ bool Player::isCloseOverSolid (float distance) const
 
 bool Player::isLanded () const
 {
-	if (!b2Vec2Equals(getLinearVelocity(), b2Vec2_zero))
```

Comment:
why is this removed here? can the player land now by sliding over the cave platform?

Reply:
Yes. No need to land perfectly from above. Now there is no need to start and land again if not.
Also before I could never land with wind present.

Reply:
so maybe we shouldn't check for zero here, but some epsilon? Otherwise the player might trigger npcs by just flying by fast - and sliding over the cave plattform.

Reply:
So far I tested, it works okay. I can fly by low. NPC will only start walking when player is reachable by platform, a check in other place.